index.css

```diff
@@ -247,15 +238,25 @@ section ul, section li {
 }
 
 .proposal-detail-label {
+    color: #aaa;
```

Comment:
This should be `#888`, to be the consistent lighter grey to contrast with `#333` on label values and the proposal titles.